pkg/queue/protobuf_stats_reporter_test.go

```diff
@@ -128,3 +120,13 @@ func TestProtoHandler(t *testing.T) {
 		})
 	}
 }
+
+func cmpStatData(t *testing.T, want metrics.Stat, got metrics.Stat) {
+	t.Helper()
+	if !cmp.Equal(want, got, ignoreStatFields) {
+		t.Errorf("Scraped stat mismatch; diff(-want,+got):\n%s", cmp.Diff(want, got))
+	}
+	if gotUptime := got.ProcessUptime; gotUptime < 5.0 || gotUptime > 6.0 {
+		t.Errorf("Got %v for process uptime, wanted 5.0 <= x < 6.0", gotUptime)
+	}
```

Comment:
Ah it's here. This makes not a ton of sense here though as it's specific to the test diluting the time by 5s. I'd just keep it in the test itself.